pennylane_lightning/src/gates/cpu_kernels/avx_common/AVXUtil.hpp

```diff
@@ -265,6 +212,40 @@ template <size_t packed_size> struct InternalWires {
 template <size_t packed_size>
 constexpr auto internal_wires_v = InternalWires<packed_size>::value;
 
+template <typename PrecisionT, size_t packed_size> struct Set;
```

Comment:
As mentioned in PR #393 
I think setvalue is an improvement alright, or even `SetIntrinsicValue`.  Happy with either of those options.  I'll also mention this in the other PR to keep it tracked.

Reply:
Updated to `setValue`, which makes more sense.